docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -864,6 +865,60 @@ https://godbolt.org/z/6qTPz9n6h
+- `none` - turns off the vtable checks.
+
+### Performance implications
+
+Caroline Tice played an important role in integrating the vtable verification (VTV) by submitting the commit of this feature in GCC 4.9. She involved on articles that give information about performance penalty. She is describe performance penalty following 3 points: 
+
+- Call verification: Performance penalty between 5% and 10%.
+- Object permission changes: Between 400% and 700% slowdown for permission changes per object file, while a performance loss of 320 ms is noticeable for permission changes per binary. 
+- Virtual function hashtable size: Storage of virtual function hashtable imply a big waste of space.
+
+Detailed information on virtual table verification and performance penalties can be found in Caroline Tice's articles.
+
+In addition to this performance penalty information, GCC's virtual table checking function is tracked by the `-fvtv-counts` flag, which provides counters to evaluate the number of virtual calls checked and the size of virtual table pointer sets for each class. These counters are useful information for assessing the performance penalty on source code. This information is written to two log files:
+
+- `vtv_count_data.log` - The number of virtual calls being verified for each class.
+- `vtv_class_set_sizes.log` - The size of the vtable pointer sets for each class.
+
+### Special considerations
+
+Virtual table verification is disabled in the default GCC configuration. GCC must be compiled with the `--enable-vtable-verify` option to enable virtual table verification. The current GCC configuration is given by `gcc -v` or `gcc -###` command.
+
+Virtual table checking can lead to failures due to incomplete data when a binary is compiled without virtual table checking but linked to libraries compiled with virtual table checking. The `libvtv_stubs` library is available for the linker to disable virtual table checking on libraries compiled with table verification flag.
+
+Additional debugging information is also available using GCC's `-fvtv-debug` flag. This flag provides information on binaries compiled with `-fvtable-verify` to gain a better understanding of the action during virtual table verification. It traces the virtual table pointers checked for each class of source code in a log file (vtv_set_ptr_data.log).
+
+The location of `-fvtv-counts` and `-fvtv-debug` log files is defined by the `VTV_LOGS_DIR` environment variable. The default location is the current directory.
+
+<!-- More information
+https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html#index-fvtable-verify
+https://gcc.gnu.org/git/?p=gcc.git;a=commitdiff;h=2077db1be5b18b94a91095a3fb380bbc4a81e61b
+https://gcc.gnu.org/wiki/cauldron2012?action=AttachFile&do=get&target=cmtice.pdf#page=38
+https://www.usenix.org/sites/default/files/conference/protected-files/sec14_slides_tice.pdf#page=14
```

Comment:
The functionality requested on the IT forum leads to this ticket: https://sir.ext.ti.com/jira/browse/EXT_EP-10096

This request has been classified as "non-priority" since November 2020 and this issue is still open with no activity. I don't think this message is relevant as it hasn't been implemented yet.